CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Add `prefix` to `string` module.
 - Add `suffix` to `string` module.
+- Add overloads to `$mock.union` function.
```

Comment:
Sorry for being picky 😬, but according to [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) this should be in `Changed`:
- `Added` for new features.
- `Changed` for changes in existing functionality.
- `Deprecated` for soon-to-be removed features.
- `Removed` for now removed features.
- `Fixed` for any bug fixes.
- `Security` in case of vulnerabilities.